Source/Blazorise/Layout.razor

```diff
@@ -3,6 +3,10 @@
 {
     <section @ref="@ElementRef" id="@ElementId" class="@ClassNames" style="@StyleNames" @attributes="@Attributes">
         <CascadingValue Value=this>
+            @if ( Loading )
+            {
+                <div class="@LoadingClass"></div>
```

Comment:
I was thinking of adding additional `[Parameter] public RenderFragment LoadingTemplate { get; set; }`, and then add it like:

```cshtml
@if ( LoadingTemplate != null )
{
    @LoadingTemplate
}
else
{
    <div class="@LoadingClass"></div>
}
```

Reply:
Ahh I see, looks good!